### ISSUE TITLE
1d_stencil seg fault

### ISSUE THREAD

**Issue by dcbdan** [STEllAR-GROUP/hpx#2182]
Running ./bin/1d_stencil_j where j is 4, 5, 6 or 7 seg faults on my machine. The OS is redhat. 

```
[dbourge]$ ./bin/1d_stencil_7
{stack-trace}: 18 frames:
0x7f84fec6b364  : hpx::termination_handler(int) + 0x124 in /home/dbourge/hpx/hpx_build_05_17/lib/libhpx.so.0
0x3d1660f7e0    : ??? + 0x3d1660f7e0 in /lib64/libpthread.so.0
0x7f84fed91b50  : ??? + 0x7f84fed91b50 in /home/dbourge/hpx/hpx_build_05_17/lib/libhpx.so.0
0x7f84fee85d80  : hpx::agas::stubs::symbol_namespace::symbol_namespace_locality(std::string const&) + 0x30 in /home/dbourge/hpx/hpx_build_05_17/lib/libhpx.so.0
0x7f84fee7524a  : hpx::agas::addressing_service::unregister_name_async(std::string const&) + 0x3a in /home/dbourge/hpx/hpx_build_05_17/lib/libhpx.so.0
0x7f84feecdc21  : hpx::agas::unregister_name_sync(std::string const&, hpx::error_code&) + 0x21 in /home/dbourge/hpx/hpx_build_05_17/lib/libhpx.so.0
0x4218b0        : ??? + 0x4218b0 in ./bin/1d_stencil_7
0x43854f        : ??? + 0x43854f in ./bin/1d_stencil_7
0x43a593        : ??? + 0x43a593 in ./bin/1d_stencil_7
0x44430b        : ??? + 0x44430b in ./bin/1d_stencil_7
0x444462        : ??? + 0x444462 in ./bin/1d_stencil_7
0x413dc9        : ??? + 0x413dc9 in ./bin/1d_stencil_7
0x416e55        : ??? + 0x416e55 in ./bin/1d_stencil_7
0x7f84fed8bf1d  : ??? + 0x7f84fed8bf1d in /home/dbourge/hpx/hpx_build_05_17/lib/libhpx.so.0
0x7f84fed8eeaf  : ??? + 0x7f84fed8eeaf in /home/dbourge/hpx/hpx_build_05_17/lib/libhpx.so.0
0x7f84fee29fb2  : hpx::threads::coroutines::detail::coroutine_impl::operator()() + 0xc2 in /home/dbourge/hpx/hpx_build_05_17/lib/libhpx.so.0
0x7f84febd3bd9  : ??? + 0x7f84febd3bd9 in /home/dbourge/hpx/hpx_build_05_17/lib/libhpx.so.0
{what}: Segmentation fault
{config}:
  HPX_HAVE_NATIVE_TLS=ON
  HPX_HAVE_STACKTRACES=ON
  HPX_HAVE_COMPRESSION_BZIP2=OFF
  HPX_HAVE_COMPRESSION_SNAPPY=OFF
  HPX_HAVE_COMPRESSION_ZLIB=OFF
  HPX_HAVE_PARCEL_COALESCING=ON
  HPX_HAVE_PARCELPORT_TCP=ON
  HPX_HAVE_PARCELPORT_MPI=OFF
  HPX_HAVE_PARCELPORT_IPC=OFF
  HPX_HAVE_PARCELPORT_IBVERBS=OFF
  HPX_HAVE_VERIFY_LOCKS=OFF
  HPX_HAVE_HWLOC=ON
  HPX_HAVE_ITTNOTIFY=OFF
  HPX_HAVE_RUN_MAIN_EVERYWHERE=OFF
  HPX_PARCEL_MAX_CONNECTIONS=512
  HPX_PARCEL_MAX_CONNECTIONS_PER_LOCALITY=4
  HPX_AGAS_LOCAL_CACHE_SIZE=4096
  HPX_HAVE_MALLOC=talloc
  HPX_PREFIX (configured)=/home/dbourge/inst_hpx
  HPX_PREFIX=/home/dbourge/hpx/hpx_build_05_17
{version}: V0.9.12-trunk (AGAS: V3.0), Git: aeafe09f23
{boost}: V1.61.0
{build-type}: release
{date}: May 18 2016 12:46:22
{platform}: linux
{compiler}: GNU C++ version 4.9.2
{stdlib}: GNU libstdc++ version 20141030
Aborted (core dumped)
```

**Comment by hkaiser** [comment]
@dcbdan This should be fixed once #2184 has been merged.

**Comment by hkaiser** [comment]
Closing this as #2184 was merged.
